xml/art_jeos_quickstart.xml

```diff
@@ -315,16 +315,16 @@
 <sect1 xml:id="sec-jeos-kvm">
   <title>Install and Run &jeos; on KVM</title>
   <para>
-    Provided you have &kvmk; installed on your local machine, you can use
+    Provided you have &kvm; installed on your local machine, you can use
```

Comment:
This sentence seems wrong? "Provided you..." ?

Reply:
It is a legitimate construct. It means:

"So long as you have KVM installed"